mpas_analysis/obs/observational_datasets.xml

```diff
@@ -0,0 +1,947 @@
+      (missing)
+    </description>
+    <source>
+      (missing)
+    </source>
+    <releasePolicy>
+      (missing)
+    </releasePolicy>
+    <references>
+      (missing)
+    </references>
+    <bibtex>
+      (missing)
+    </bibtex>
+    <dataUrls>
+      (missing)
+    </dataUrls>
+    <preprocessing>
+      (missing)
+    </preprocessing>
+    <tasks>
+      - indexNino34
+    </tasks>
+  </observation>
+
+  <observation>
+    <name>
+      ERS SSTv4 Nino 3.4 Index
+    </name>
+    <component>
+      ocean
+    </component>
+    <description>
+      (missing)
+    </description>
+    <source>
+      (missing)
+    </source>
+    <releasePolicy>
+      (missing)
+    </releasePolicy>
+    <references>
+      (missing)
+    </references>
+    <bibtex>
+      (missing)
+    </bibtex>
+    <dataUrls>
+      (missing)
+    </dataUrls>
+    <preprocessing>
+      (missing)
+    </preprocessing>
+    <tasks>
+      - indexNino34
+    </tasks>
+  </observation>
+
+<!--
+  <observation>
+    <name>
+    </name>
+    <component>
+      ocean
+    </component>
+    <description>
+    </description>
+    <source>
+      []()
+    </source>
+    <releasePolicy>
+    </releasePolicy>
+    <references>
+      -
+    </references>
+    <bibtex>
+    </bibtex>
+    <dataUrls>
+      -
+    </dataUrls>
+    <preprocessing>
+    </preprocessing>
+    <tasks>
+      -
+    </tasks>
+  </observation>
+-->
+
+  <!-- SEA ICE -->
+
+  <observation>
+    <name>
+      Ice concentration: SSM/I, NASATeam algorithm
+    </name>
+    <component>
+      seaice
+    </component>
+    <description>
+      This data set is generated from brightness temperature data and is
+      designed to provide a consistent time series of sea ice concentrations
+      spanning the coverage of several passive microwave instruments. The data
+      are provided in the polar stereographic projection at a grid cell size of
+      25 x 25 km.
+    </description>
+    <source>
+      [NSIDC NASATeam Website](http://nsidc.org/data/NSIDC-0051)
+    </source>
+    <releasePolicy>
+      NASA data are not copyrighted; however, when you publish our data or
+      results derived therefrom, we request that you include an acknowledgment
+      within the text of the publication and reference list.
+      [Data Citation and Acknowledgements]
+      (https://podaac.jpl.nasa.gov/CitingPODAAC)
+    </releasePolicy>
+    <references>
+      [Cavalieri et al. (1996)](https://doi.org/10.5067/8GQ8LZQVL0VL)
+    </references>
+    <bibtex>
+      @misc{Cavalieri1996,
+      author = {D. J. Cavalieri,  and C. L. Parkinson and P. Gloersen and H. J. Zwally},
+      title = {Sea Ice Concentrations from Nimbus-7 SMMR and DMSP SSM/I-SSMIS Passive Microwave Data, Version 1},
+      year = {1996},
+      doi = {10.5067/8GQ8LZQVL0VL},
+      notes = {updated yearly},
+      url = {https://doi.org/10.5067/8GQ8LZQVL0VL}
+      }
+    </bibtex>
+    <dataUrls>
+      (missing, requires registration with NSIDC)
```

Comment:
@milenaveneziani, can you help me figure out what the URL would be here?

Reply:
hmm, look at this:
https://urs.earthdata.nasa.gov/oauth/authorize?client_id=OeZxXA9CSNdsGKSltLUmIQ&response_type=code&redirect_uri=https%3A%2F%2Fdaacdata.apps.nsidc.org%2Foauth%2Fredirect_uri&state=aHR0cHM6Ly9kYWFjZGF0YS5hcHBzLm5zaWRjLm9yZy9wdWIvREFUQVNFVFMvbnNpZGMwMDUxX2dzZmNfbmFzYXRlYW1fc2VhaWNlLw
looks like a registration is needed now?? I am confused..

Reply:
Yes, I'm registered but then I just see a huge number of `.bin` files, one for each month since 1979.  There obviously was also a pre-processing script if this is what the input data was...

Reply:
Yes, the sea-ice data is the one we (myself and @maltrud) pre-processed the most. We went from binary to netcdf files, and from the polar stereographic projection to the regular lon/lat grid. I did the first step, with a matlab script, and Mat did the second step, either with feret or ncremap, now I can't recall.
But I didn't know now a user registration was necessary..

Reply:
same for ICESat.

Reply:
Okay, well, we're going to have to redo that stuff in a python script.  Wen we do that, we might want to consider staying with the polar stereographic grid like I already do for the Antarctic melt and SOSE plots (i.e. leaving the obs on that grid and just remapping MPAS data to the same grid).  But clearly that's way beyond the scope of the current database.

Reply:
Added all scripts I have into the preprocessing_obs folder. I also added a note txt file with reminders about how we preprocessed the single seaice data sets.